config/notifiers.go

```diff
@@ -870,6 +883,73 @@ func (c *JiraConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	if c.IssueType == "" {
 		return fmt.Errorf("missing issue_type in jira_config")
 	}
+	return nil
+}
+
+type RocketchatAttachmentField struct {
+	Short *bool  `json:"short"`
+	Title string `json:"title,omitempty"`
+	Value string `json:"value,omitempty"`
+}
+
+const (
+	ProcessingTypeSendMessage        string = "sendMessage"
```

Comment:
Don't need `string` here, can use type inference. Sorry, this wasn't clear last time.

```suggestion
	ProcessingTypeSendMessage = "sendMessage"
```